src/cmd/policy.go

```diff
@@ -54,6 +62,15 @@ opslevel run policy -f policy.rego | jq
 	},
 }
 
+func RegoFuncReadFile(ctx rego.BuiltinContext, a *ast.Term) (*ast.Term, error) {
+	if str, ok := a.Value.(ast.String); ok {
+		contents, err := os.ReadFile(string(str))
```

Comment:
I know there's different ways to read a file, e.g. line-by-line, chunks, etc.  My understanding is chunks would be better in the case of larger files so I can update this if we'd prefer to read it that way.  I can focus on that after lunch if so.

Reply:
Chunking is best if you are going to be reading through the file and process data as it comes in reusing the buffer as you move through the data.  Unfortunately because we want to load up the contents of the file fully and then have Rego run more policy stuff on the content we need to fully load the contents into more - so this is the best way.

Reply:
Ok, that was one of my concerns. I wasn't sure how the content would look on the Rego side.